vietnamese/wca-guidelines.md

```diff
@@ -80,7 +80,7 @@ Mỗi quy định được phân loại bằng một trong các nhãn sau để
 - 3h++) [VÍ DỤ] Ví dụ của việc khả năng hoạt động cơ bản bị ảnh hưởng: có thể xoay bước mới, không thể xoay bước bình thường, có nhiều viên hoặc mặt ở trong tầm nhìn hơn so với khối hình gốc, các màu ở mặt sau khối hình ở trong tầm nhìn, khối hình có thể tự động xoay các bước, hoặc khối hình có nhiều trạng thái giải hoặc có trạng thái giải khác với khối hình gốc.
 - 3h2++) [LÀM RÕ] Các khối hình "stickerless" khác biệt rõ rệt so với các khối hình "stickerless" được sản xuất thông thường chỉ được phép sử dụng khi có sự cho phép của WCA Delegate.
 - 3h2a+) [BỔ SUNG] Ví dụ các bộ phận trong suốt không cung cấp thêm thông tin về trạng thái khối hình: bộ phận phía bên trong khối hình bằng nhựa trong suốt, vỏ ngoài Clock bằng nhựa trong suốt.
-- 3j++) [VÍ DỤ] Hướng dẫn Trực quan (bằng tiếng Anh) của [Quy định 3j](regulations:regulation:3j) ở tại [link này](https://drive.google.com/file/d/1m6THsA8fXRN7QFM4ApJbm6eVODKGbMLx/view).
+- 3j++) [VÍ DỤ] Hướng dẫn Trực quan (bằng tiếng Anh) của [Quy định 3j](regulations:regulation:3j) ở tại [link này](https://drive.google.com/file/d/11kSxoCgJ2i0V1Jx9wtpNNFDirdTHL5s5/view).
```

Comment:
```suggestion
- 3j++) [VÍ DỤ] Hướng dẫn Trực quan của [Quy định 3j](regulations:regulation:3j) ở tại [link này](https://drive.google.com/file/d/11kSxoCgJ2i0V1Jx9wtpNNFDirdTHL5s5/view).
```
The text in brackets said "in English" as it was linking to the original English version, now that there is a translation we won't need that